packages/application-extension/src/index.tsx

```diff
@@ -362,6 +366,61 @@ const mainCommands: JupyterFrontEndPlugin<void> = {
         isEnabled: () => !labShell.isEmpty('right')
       });
 
+      commands.addCommand(CommandIDs.activateSidebarOverlays, {
+        label: trans.__('Show Side bar overlays'),
+        execute: args => {
+          console.log('correct');
+          if (args.side != 'left' && args.side != 'right') {
+            throw Error(`Unsupported sidebar: ${args.side}`);
+          }
+
+          const widgets = Array.from(labShell.widgets(args.side));
+          for (let index = 0; index < widgets.length; index++) {
+            let elementId = widgets[index].id;
+            let focusElement = document.querySelector(
+              "[data-id='" + elementId + "']"
+            );
+
+            if (focusElement && !focusElement.classList.contains('overlay')) {
+              const options: Overlay.IOptions = {
+                hostElement: focusElement,
+                classConfig: {
+                  class: 'JP-Overlay',
+                  title: 'JP-Overlay-Tooltip'
+                },
+                cssConfig: {
+                  position: 'absolute',
+                  top: '67%',
+                  left: '45%',
+                  width: '47%',
+                  height: '32%',
+                  'border-Radius': '25%',
+                  'z-index': 'inherit',
+                  'text-align': 'center',
+                  color: 'rgb(255, 255, 255)',
+                  background: 'rgb(51, 51, 51)',
+                  visibility: 'hidden'
```

Comment:
This should be in a CSS file instead. We have linting setup for CSS files which protects against invalid values; further this allows themes to customize. Setting CSS in code does neither

Reply:
I have moved this to base.css.